docs/repositories-configure/adjusting-quality-settings.md

```diff
@@ -27,7 +27,7 @@ The following screenshot displays the default configuration values:
 -   **New security issues are over:** Pull requests or commits are marked not up to standards if the number of security issues introduced is higher than the set value.
 -   **Complexity is over:** Pull requests or commits are marked not up to standards if the introduced complexity is higher than the set value.
 -   **Duplication is over:** Pull requests or commits are marked not up to standards if the number of clones introduced is higher than the set value.
--   **Diff coverage is under:** Pull requests are marked not up to standards if the diff coverage of the pull request is lower than the set value. This rule is only available for pull requests.
+-   **Diff coverage is under:** Pull requests are marked not up to standards if the diff coverage of the pull request is applicable (different from `∅`) and lower than the set value. This rule is only available for pull requests.
```

Comment:
I'm thinking that mentioning the Null here convolutes the explanation of what the gate does. Makes it harder to understand how the gate works. I would wonder what "applicable" means and what use-cases cause that.

Reply:
Goo point, I was also unsure if I should mention it here or not. But I would also like to mention somewhere that the `∅` value doesn't make the quality gate fail. :thinking: Do you think this is relevant? Or that perhaps, since this is actually the expected thing to happen, it's not worth mentioning it for now?

Reply:
I tend to agree, but at the same time, it would be nice to have some kind of place with this info, like you just did. So for me, we can either leave it as it is, or consider linking the `applicable` to the place where we explain the `**Diff coverage:** Code coverage of the coverable lines added or changed by the pull request, or `∅` if no coverable lines were changed`

Reply:
I think that both your feedback is valid.

For now, I decided to simplify a bit the description of the diff coverage quality gate, while still mentioning that the value `∅` also passes the gate, and including a link on "not applicable" to the section where we define the code coverage metrics.

Tell me what you think, as we still have some time to reconsider this until we release the feature.